spec/controllers/queue_items_controller_spec.rb

```diff
@@ -3,200 +3,173 @@
 describe QueueItemsController do
   describe 'GET index' do
     it 'sets @queue_items with authentication' do
-      user = Fabricate(:user)
-      session[:user_id] = user.id
-      queue_items = user.queue_items
+      set_current_user
```

Comment:
You can put set_current_user even way above - a line after `describe QueueItemsController do` because you will be requiring user in every action ,right.